web-components/src/components/phone-input/PhoneInput.ts

```diff
@@ -227,6 +228,7 @@ export namespace PhoneInput {
             @change-selected="${(e: CustomEvent) => this.handleCountryChange(e)}"
             clear-icon-height="${this.clearIconHeight}"
             with-custom-content
+            clear-aria-label="${this.clearAriaLabel}"
```

Comment:
what is clear-aria-label?

Reply:
by default  add english value in  clear-aria-label if value is not passed by client code

Reply:
> what is clear-aria-label?

clear-aria-label is the attribute that combobox is expecting. Also combobox has the default set for this attribute.
Will add the default value here too.

Reply:
default value added